internal/handlers/common/msg_listcommands.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+	"sort"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/FerretDB/FerretDB/internal/handlers"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// command represents a handler command.
+type command struct {
+	// Help is shown in the help function
+	Help string
+
+	// Handler processes command
+	Handler func(handlers.Interface, context.Context, *wire.OpMsg) (*wire.OpMsg, error)
+}
+
+// Commands is a map of Commands that Handler interface can support.
+// Order of entries matches the interface definition.
+//
+// Please keep help text in sync with handlers.Interface methods documentation.
+var Commands = map[string]command{
+	// sorted alphabetically
+	"buildInfo": {
+		Help:    "Returns a summary of the build information.",
+		Handler: (handlers.Interface).MsgBuildInfo,
+	},
+	"collStats": {
+		Help:    "Returns storage data for a collection.",
+		Handler: (handlers.Interface).MsgCollStats,
+	},
+	"connectionStatus": {
+		Help: "Returns information about the current connection, " +
+			"specifically the state of authenticated users and their available permissions.",
+		Handler: (handlers.Interface).MsgConnectionStatus,
+	},
+	"count": {
+		Help:    "Returns the count of documents that's matched by the query.",
+		Handler: (handlers.Interface).MsgCount,
+	},
+	"create": {
+		Help:    "Creates the collection.",
+		Handler: (handlers.Interface).MsgCreate,
+	},
+	"createIndexes": {
+		Help:    "Creates indexes on a collection.",
+		Handler: (handlers.Interface).MsgCreateIndexes,
+	},
+	"dataSize": {
+		Help:    "Returns the size of the collection in bytes.",
+		Handler: (handlers.Interface).MsgDataSize,
+	},
+	"dbStats": {
+		Help:    "Returns the statistics of the database.",
```

Comment:
With the `<noun> of <noun>` the sentence sounds harder to read/speak. Instead, it's suggested to use `<adj> <noun>` here and after.

```suggestion
		Help:    "Returns the database statistics.",
```